Dockerfile.template

```diff
@@ -58,6 +58,12 @@ RUN set -eux; \
 		xfslibs-dev \
 		zlib1g-dev \
 	; \
+	wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add - ; \
+	echo "deb https://download.ceph.com/debian-octopus buster main" >/etc/apt/sources.list.d/ceph.list ; \
```

Comment:
I'm curious -- why pull this package from Ceph's repos instead of just using the Debian package?  Also, wouldn't we need the relevant `-dev` package to get the headers?

Reply:
(I ask because for me, installing this package from the Ceph repos is really a non-starter, to be explicit, so I'd like to understand why it's necessary.)

Reply:
The version from the Debian repository is very old (luminous) and, while it's still being maintained, doesn't allow you to use qemu against a newer cluster (such as octopus):
```root@f83b2c0b5c28:/# apt-cache policy librbd1
librbd1:
  Installed: (none)
  Candidate: 15.2.4-1~bpo10+1
  Version table:
     15.2.4-1~bpo10+1 500
        500 https://download.ceph.com/debian-octopus buster/main amd64 Packages
     12.2.11+dfsg1-2.1+b1 500
        500 http://deb.debian.org/debian buster/main amd64 Packages
```
You are right about `librbd-dev` though. I'll modify the PR. This will probably resolve https://github.com/tianon/docker-qemu/pull/11/files#r478703528 as well

Reply:
Another reason for using the current (octopus) release: Ceph has a setting called `require_min_compat_client`. If it's set to anything newer than what Debian has in the repository - Luminous in this case,  the client might be prevented to connect to the Ceph cluster

Reply:
An alternative would be switching to `ubuntu:focal` instead of using a debian base. At the time of this writing, version `15.2.3` is in Ubuntu.

Please let me know if that would be acceptable - I can prepare a MR.

Reply:
I'd rather stick with Debian -- is the version in `buster-backports` new enough?  If not, what would it take to get it updated in unstable so it could be backported?

Reply:
For my particular use case, the version in unstable is too old. However, I can build myself a modified `start-qemu` script that covers that.